### PR TITLE
small style/typo edits to readmes

### PR DESCRIPTION
**Pull request by BarishNamazov**
label should be closed by label

### CODE REVIEW

File: set-data/README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ but you can customize how (or even if) it’s displayed:
 ```html
 <v-app data='{"c": 25}'>
 	<input v-model="c" type="number" /> ℃ =
-	<set-data name="f" :value="c * 1.8 + 32">{{ +f.toFixed(2) }} ℉</set-data>
+	<set-data name="f" :value="c * 1.8 + 32">{{ f.toFixed(2) }} ℉</set-data>
```
Comment (DmitrySharabin) [inline set-data/README.md:46]: I believe  `+f.toFixed(2)` is crucial here: `f` is of type `string`, and there is no `toFixed()` method on strings. So we need to convert it to a number either with `Number(f)`, or with `+f`.

File: v-app/README.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ Custom methods:
 	// or document.querySelector("#sum_demo").methods = ...
 	sum_demo.methods = {
 		sum (numbers) {
-			return numbers.reduce((a, c) => a + +c, 0);
+			return numbers.reduce((a, c) => a + c, 0);
```
Comment (DmitrySharabin) [inline v-app/README.md:83]: And we can't get rid of `+` here for the same reason.

File: set-data/README.md (hunk 1 above)
Comment (BarishNamazov) [reply set-data/README.md:46]: Ah, I see, thanks. For readability reasons, I think `Number(f)` or `parseFloat(f)` would be better.

**Comment by BarishNamazov** [comment]
I reverted the last commit, so I think you should be able to merge now if it looks good.

File: set-data/README.md (hunk 1 above)
Comment (LeaVerou) [reply set-data/README.md]: I agree that `Number(f)` is more readable (but definitely not `parseFloat()`)

**Comment by LeaVerou** [comment]
I see that the tag typo was fixed here: https://github.com/mavoweb/mavue/commit/463d9489e691e59692e4ce0f8ab398cee0a4c98a
Should this PR be closed or rebased & merged?

**Pull request closed without merging**